packages/@aws-cdk/aws-elasticloadbalancing/README.md

```diff
@@ -44,3 +44,6 @@ lb.addListener({
     allowConnectionsFrom: [mySecurityGroup]
 });
 ```
+
+The `sslCertificateId` property of the `LoadBalancerListener` class has been deprecated.
+It is recommended to use `sslCertificateArn` property instead.
```

Comment:
I think we can skip the README update in this case, as it describes a change more so than the current state. Can you revert these changes?

Reply:
Do I need to do anything else here, because a check is failing due to there not being changes to the README.md?